CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+vNext (TBD)
+------------------
+
+The sync protocol version has been bumped to version 25. The server is backwards-compatible with clients using protocol version 24 or below, but clients at version 25 are not backwards-compatible with a server at protocol version 24. The server must be upgraded before any clients are upgraded.
```

Comment:
I wouldn't refer to the sync protocol version - it's not a known number. What users know is the ROS version.
You could take inspiration for the text for e.g. realm-js:

You will need to upgrade your Realm Object Server to at least version 3.11.0 or use Realm Cloud. If you try to connect to a ROS v3.10.x or previous, you will see an error like Wrong protocol version in Sync HTTP request, client protocol version = 25, server protocol version = 24.